docs/workflow.md

```diff
@@ -129,7 +129,26 @@ $ kubecfg diff kubeprod-manifest.jsonnet
 $ kubecfg update --ignore-unknown=true --gc-tag kube_prod_runtime kubeprod-manifest.jsonnet
 ```
 
-NOTE: There is a [bug in kubecfg](https://github.com/ksonnet/kubecfg/issues/211) that always requires using `--ignore-unknown=true`, even if this command-line flag defaults to `true`.
+The `kube_prod_runtime` garbage collection tag specified in the `kubecfg update` command takes care of garbage collection to ensure there is no leakage of Kubernetes resources.
+
+> **NOTE**
+> A [bug in kubecfg](https://github.com/ksonnet/kubecfg/issues/211) requires that the `--ignore-unknown=true` flag always be specified.
```

Comment:
I believe the `--ignore-unknown=true` is only required when installing BKPR for the first time, but not for successive updates.

Reply:
It might be true for future updates too, when we add something that involves additional custom types (eg: prometheus-operator).

It is reasonable to just always provide `--ignore-unknown=true`, and indeed this is actually _meant_ to be the kubecfg default, I just botched the PR that added that flag :(